Ignore/LoginPage.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
In the interest of keeping the code clean, let's not commit anything that shouldn't be part of the project.